futures-channel/src/mpsc/mod.rs

```diff
@@ -88,7 +88,7 @@ use std::usize;
 use futures_core::task::{self, Waker};
 use futures_core::{Poll, Stream};
 
-use mpsc::queue::{Queue, PopResult};
+use crate::mpsc::queue::{Queue, PopResult};
```

Comment:
Is this what `rustfix` generates? I think `use self::queue::{....};` makes more sense than walking down from the crate root.

Reply:
Yes. This is what was generated. However, the line it replaced seems to have used a crate relative path as well.